src/Misc/externals.sh

```diff
@@ -5,8 +5,10 @@ PRECACHE=$2
 NODE_URL=https://nodejs.org/dist
 UNOFFICIAL_NODE_URL=https://unofficial-builds.nodejs.org/download/release
 NODE_ALPINE_URL=https://github.com/actions/alpine_nodejs/releases/download
+# When you update Node versions you must also create a new release of alpine_nodejs at that updated version.
+# Follow the instructions here: https://github.com/actions/alpine_nodejs?tab=readme-ov-file#getting-started
 NODE16_VERSION="16.20.2"
-NODE20_VERSION="20.8.1"
+NODE20_VERSION="20.13.1"
 # used only for win-arm64, remove node16 unofficial version when official version is available
```

Comment:
does win-arm64 not having node20?

Reply:
No, they've got a release for `win-arm64`: https://nodejs.org/dist/v20.13.1/

Reply:
Let's move that comment to the end of the line so it's more obvious which variable it's referring to: 
25032602a34c4dcef76491e1fdf6d8e916fee6f4